src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/ValidatorTest.java

```diff
@@ -343,19 +383,6 @@ public void importAndFunctionShouldNotSkipParsing() throws Exception {
     }
 
     private void failWithError(final String... errors) throws Exception {
```

Comment:
Should be called `assertFailWithError` it took me a while to get why you just failed the test with a message when you actually made an assertion that the build would fail with a specific message.